package.json

```diff
@@ -6,6 +6,9 @@
   "repository": "muut/riotjs",
   "author": "Muut, Inc. and other contributors",
   "license": "MIT",
+  "scripts": {
+    "test": "make test"
```

Comment:
I have deleted the .travis file but the `npm run test` could stay, it does not disturb anyone